src/content/en/ai-capabilities/subject-transfer.md

```
@@ -0,0 +1,97 @@
+
+The big problem is that **training is required**. There is no casualness.
+
+---
+
+## image2prompt
+
+As the most primitive method, there is a method of "generating a caption from an image and running text2image with that caption."
+
+You might think, "With such a primitive method?" but it is theoretically possible if there is an MLLM that can perfectly describe the reference image and an image generation model that can perfectly reproduce that description.
+
+![](https://gyazo.com/26351f2e5d3eb17c623acd815ba8709c){gyazo=image}
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)